test/tools/runner/config.ts

```diff
@@ -205,6 +208,11 @@ export class TestConfiguration {
 
   newClient(urlOrQueryOptions?: string | Record<string, any>, serverOptions?: MongoClientOptions) {
     serverOptions = Object.assign({}, getEnvironmentalOptions(), serverOptions);
+
+    if (this.loggingEnabled && !Object.hasOwn(serverOptions, 'mongodbLogPath')) {
+      serverOptions = this.setupLogging(serverOptions);
```

Comment:
Can we add some sort of unique identifier to the logs?  If a test has multiple clients, they'll each enable logging and all the logs will intersperse, making it hard to determine what is actually happening per client.

Reply:
Probably something we should think about for our actual logging system, I recall it being discussed on the spec. Anyway, yea we can throw on an objectId

Reply:
The entity name only is relevant for unified tests though?  What about non-unified tests?

Reply:
added a hex string for when the name is omitted